src/derivers/bip39.test.ts

```diff
@@ -4,16 +4,85 @@ import {
   concatBytes,
   hexToBytes,
 } from '@metamask/utils';
-import * as hmacModule from '@noble/hashes/hmac';
 
 import fixtures from '../../test/fixtures';
+import * as cryptography from '../cryptography';
 import { secp256k1, ed25519Bip32, type Curve } from '../curves';
+import { mnemonicPhraseToBytes } from '../utils';
 import {
   entropyToCip3MasterNode,
   createBip39KeyFromSeed,
   deriveChildKey,
+  mnemonicToSeed,
 } from './bip39';
 
+const TEST_MNEMONIC_PHRASE =
+  'pill frown erosion humor invest inquiry rich garment seek such mention punch';
+
+describe('mnemonicToSeed', () => {
+  describe('without passphrase', () => {
+    const seed = new Uint8Array([
```

Comment:
What's the source of truth for this seed that we are comparing with?

Reply:
https://github.com/MetaMask/scure-bip39/blob/612c6952ca8aee034e32dd0dc307c1d96e325ae2/test/bip39.test.ts#L127-L132

Reply:
Let's reference that as a comment for our future selves 😄